app/src/main/java/co/nimblehq/compose/crypto/ui/screens/home/HomeScreen.kt

```diff
@@ -37,18 +47,55 @@ fun HomeScreen() {
                 text = stringResource(id = R.string.home_title),
                 textAlign = TextAlign.Center,
                 style = Style.semiBold24(),
-                color = MaterialTheme.colors.titleHome
+                color = MaterialTheme.colors.textColor
             )
 
             PortfolioCard(
                 modifier = Modifier.constrainAs(portfolioCard) {
                     top.linkTo(title.bottom, margin = Dp40)
                 }
             )
+
+            Text(
+                modifier = Modifier.constrainAs(myCoinsTitle) {
+                    top.linkTo(portfolioCard.bottom, margin = Dp52)
+                    start.linkTo(parent.start)
+                    width = Dimension.preferredWrapContent
+
+                },
+                text = stringResource(id = R.string.home_my_coins_title),
+                style = Style.medium16(),
+                color = MaterialTheme.colors.textColor
+            )
+
+            SeeAll(
+                modifier = Modifier
+                    .clickable(onClick = { /* TODO: Update on Integrate ticket */ })
+                    .constrainAs(seeAll) {
+                        top.linkTo(myCoinsTitle.top)
+                        end.linkTo(parent.end)
+                        width = Dimension.preferredWrapContent
+                    }
+            )
+
+            LazyRow(
```

Comment:
I think we need to add content padding here. The list items should be scrolled horizontally through the screen edges.

```suggestion
            LazyRow(
                contentPadding = PaddingValues(horizontal = Dp16),
```

https://user-images.githubusercontent.com/16315358/184799641-0118f4db-f039-47e0-b680-bbb39a752473.mp4